### PR TITLE
Add form on the first page

### PR DESCRIPTION
**Pull request by berlinum**
Add form on the first page

### CODE REVIEW

File: src/components/Form.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  background-color: #e1e1e3;
+  border: none;
+  border-radius: 10px;
+  font-family: SF;
+  font-size: 20px;
+  padding-left: 20px;
+  box-sizing: border-box;
+  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.2);
+  /* outline: orange solid;
+  border-radius: 10px; */
```
Comment (marwinburesch) [inline src/components/Form.css]: pls no commented-out code unless you are taking notes in a WIP or writing a tutorial or exercise

File: src/components/Form.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+}
+.question__label {
+  align-self: flex-start;
+  margin-left: 34px;
+  font-size: 30px;
+  font-family: SF;
+  color: #707070;
+}
+.question__input {
+  box-sizing: border-box;
```
Comment (marwinburesch) [inline src/components/Form.css]: there's an easier and more efficient way for declaring this for every element :) remember

**Comment by berlinum** [comment]
Also add outline style

File: src/components/Form.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  width: 310px;
+  height: 45px;
+  background-color: #e1e1e3;
+  border: none;
+  border-radius: 23px;
+  font-family: SF;
+  font-size: 20px;
+  padding-left: 29px;
+  margin: 19px 32px 0 33px;
```
Comment (marwinburesch) [inline src/components/Form.css]: can't you just use the `flexbox` to position this child element?

File: src/components/Header.css [hunk 4]
```diff
@@ -1,16 +1,17 @@
 .header {
   width: 100%;
-  height: 136px;
+  height: 40vw;
```
Comment (marwinburesch) [inline src/components/Header.css]: sweeeeet!

**Pull request merged**